ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -97,11 +92,11 @@ const CancelSpeedupPopover = () => {
     >
       <AppLoadingSpinner className="cancel-speedup-popover__spinner" />
       <div className="cancel-speedup-popover__wrapper">
-        <Typography
-          boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
-          variant={TypographyVariant.H6}
-          marginTop={0}
-          marginBottom={2}
+        <Text
+          variant={TextVariant.bodySm}
+          className="cancel-speedup-popover__description"
+          // gets passed through to Box component
+          display={DISPLAY.FLEX}
         >
           {t('cancelSpeedUpLabel', [
             <strong key="cancelSpeedupReplace">{t('replace')}</strong>,
```

Comment:
Note to @garrettbear we should support `strong` tags I've created an issue https://github.com/MetaMask/metamask-extension/issues/18651